src/sections/header.liquid

```diff
@@ -0,0 +1,581 @@
+    --menu-bottom-border-top-width-desktop: 0;
+    --menu-bottom-padding-top-desktop: 0;
+    --menu-bottom-width-desktop: auto;
+    --menu-margin-x-desktop: auto;
+    --menu-indent-desktop: 0 20px;
+    --order-app-desktop: 3;
+    --order-button-desktop: 6;
+    --order-cart-desktop: 5;
+    --order-logo-desktop: 1;
+    --order-menu-desktop: 2;
+    --order-search-desktop: 4;
+  {%- else -%}
+    --menu-bottom-border-top-width-desktop: 100vw;
+    {%- if header_outline == "bordered" -%}
+      --menu-bottom-margin-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %};
+      --menu-bottom-border-top-width-desktop: calc(100% + var(--menu-bottom-margin-desktop) * 2);
+    {%- endif -%}
+    --menu-bottom-padding-top-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %} 0 0;
+    --menu-margin-x-desktop: -22px;
+    --menu-indent-desktop: {% if padding_bottom_desktop == 'lg' %}20px{% elsif padding_bottom_desktop == 'md' %}16px{% else %}12px{% endif %} 0 0;
+  {%- endif -%}
+
+  {%- if menu_alignment_horizontal == 'left' -%}
+    --menu-horizontal-position-desktop: flex-start;
+  {%- elsif menu_alignment_horizontal == 'right' -%}
+    --menu-horizontal-position-desktop: flex-end;
+  {%- endif -%}
+{%- endcapture -%}
+{% comment %} CSS variables end {% endcomment %}
+
+{%- if show_mini_cart -%}
+  {%- capture element_mini_cart -%}
+    <div class="{{- class -}}__cart">
+      {% comment %} CSS variables start {% endcomment %}
+      {%- render 'css-mini-cart-button' -%}
+      {% comment %} CSS variables end {% endcomment %}
+
+      {{- cart_button -}}
+      <i class="fa-regular fa-spinner-third fa-spin"></i>
+    </div>
+  {%- endcapture -%}
+{%- endif -%}
+
+{%- for block in section.blocks -%}
+  {%- case block.type -%}
+    {%- when 'logo' -%}
+      {%- capture element_logo -%}
+        {%- assign logo         = block.settings.logo_image -%}
+        {%- assign logo_floated = block.settings.logo_image_floated -%}
+        {%- assign logo_title   = block.settings.logo_custom_title -%}
+        {%- assign svg          = block.settings.logo_svg -%}
+        {%- assign svg_floated  = block.settings.logo_svg_floated -%}
+
+        {%- render 'logo'
+            logo: logo,
+            logo_floated: logo_floated,
+            logo_title: logo_title,
+            routes: routes,
+            section_class: class,
+            shop: shop,
+            svg: svg,
+            svg_floated: svg_floated,
+            transparency: show_transparency
+        -%}
+      {%- endcapture -%}
+
+    {%- when 'menu' -%}
+      {%- assign menu            = block.settings.menu -%}
+      {%- assign menu_back_label = block.settings.menu_back_label -%}
+
+      {%- if menu != blank -%}
+        {%- if app -%}
+          {%- capture element_app_mobile -%}
+            <div class="{{- class -}}__app">
```

Comment:
Why is `class` here a variable? Can we just do `<div class="header__app">` instead?